kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -358,13 +359,12 @@ function showLearnChannel(store, channelId, cursor) {
               next: allContentCollection.next,
               previous: allContentCollection.previous,
             },
+            channelTitle: currentChannelTitle,
           };
           store.dispatch('SET_PAGE_STATE', pageState);
           store.dispatch('CORE_SET_PAGE_LOADING', false);
           store.dispatch('CORE_SET_ERROR', null);
-
-          const currentChannel = coreGetters.getCurrentChannelObject(store.state);
-          store.dispatch('CORE_SET_TITLE', `Learn - ${currentChannel.title}`);
+          store.dispatch('CORE_SET_TITLE', `Learn - ${currentChannelTitle}`);
```

Comment:
Note, we can now translate page titles, we should clean this up before releasing 0.6.x!